dig.go

```diff
@@ -198,6 +199,18 @@ func FillProvideInfo(info *ProvideInfo) ProvideOption {
 	})
 }
 
+// LocationForPC is a ProvideOption which specifies an alternate function program
+// counter address to be used for debug information. The package, name, file and
+// line number of this alternate function address will be used in error messages
+// and DOT graphs. This option is intended to be used with functions created
+// with the reflect.MakeFunc method whose error messages are otherwise hard to
+// understand
+func LocationForPC(funcPc uintptr) ProvideOption {
```

Comment:
Can we just call the argument `pc` instead of `funcPc`? I think it's a little redundant to call this "FuncPc" since `pc` already encapsulates the info that this should be pointing to a func.

```suggestion
func LocationForPC(pc uintptr) ProvideOption {
```